CHANGELOG.md

```diff
@@ -116,6 +116,15 @@ You can either directly check the reported code or verify with our documentation
 For a complete list of behavioural changes in our public and internal API, you can consult our API stability
 regression test suite and patches at https://github.com/seqan/seqan3/tree/master/test/api_stablility/3.0.2/.
 
+#### Alignment
+
+* The member constants of `seqan3::aminoacid_similarity_matrix` were changed to lower case
+  ([\#....](https://github.com/seqan/seqan3/pull/....)):
```

Comment:
```suggestion
  ([\#2599](https://github.com/seqan/seqan3/pull/2599)):
```

Reply:
Yes, I marked it for me to make the change when I received your review. Sorry for the confusion.

Reply:
I guess it's for when other changes are applied